ROMFS/px4fmu_common/init.d/4100_tiltquadrotor

```diff
@@ -0,0 +1,35 @@
+#!nsh
+#
+# @name Tilt-Quadrotor
+#
+# @url http://www.alivaero.com/the-project.html
+#
+# @type Tilt-Quad
+# @class Copter
+#
+# @output MAIN1 motor 1
+# @output MAIN2 motor 2
+# @output MAIN3 motor 3
+# @output MAIN4 motor 4
+# @output AUX1 Outer servo motor for rotor 2 arm
+# @output AUX2 Outer servo motor for rotor 4 arm
+# @output AUX3 Inner servo motor for rotor 2 arm
+# @output AUX4 Inner servo motor for rotor 4 arm
+#
+# @maintainer Ricardo Marques <marques.ricardo17@gmail.com>
+#
+
+sh /etc/init.d/rc.mc_defaults
+
+#Parameters here:
+param set LED_RGB_MAXBRT 8
```

Comment:
You may want to put parameters like this behind an AUTOCNF conditional. Example https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/13013_deltaquad#L22

That way they're only applied once (at initial airframe selection), but then the user is free to adjust without it being reset every bootup.

Reply:
Good shout. I’ll update it tomorrow.